app/models/piece.rb

```diff
@@ -7,9 +7,39 @@ def move_piece(new_x, new_y)
     update_attributes(x_cord: new_x, y_cord: new_y)
   end
 
-	def is_obstructed?(x, y)
```

Comment:
somehow the logic seems to be the same and the test is failing like before =/

I'll keep hacking!  @catherinezzy let's work off of my branch because it's already up to date with master. I just copied your code so we'd be working off the same thing.